### DISCUSSION TITLE
Open Source

### DISCUSSION THREAD

**Discussion by jesterb0206** [community/community#43189] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a repository that I opened Issues in that I would like some help with from other developers on GitHub. Does anyone know of any good repositories where I can submit my repo for open source help?

**Comment by diva-21** [comment]
Hey @jesterb0206 ,
One thing by opening issues on your reposistary will make us to look upon.
But the thing is how ppl can help you,thats the main problem. You can better use Twitter /Discord to seek help and you may get good developers to solve issues. 
Most them connect in Twitter, so i would recommend you that by which you can get ppl/ repos /developers to help you

Thank You

**Reply by jesterb0206** [reply]
@diva-21 Thank you for your response, I appreciate it!

**Comment by jfullstackdev** [comment]
yes, it's using social media to gather them 

since you already posted it here, you can post the link of your repo, so we can check too

**Reply by jesterb0206** [reply]
@jdevstatic I think you answered the last question I posed in general as well, thank you again!

Here's the link to my repository:

https://github.com/jesterb0206/natours

I think the biggest thing that frustrates me right now is the fact that my API looks terrible on mobile devices and I still need to implement a sign up form (those are the more pressing issues, but there's a few others as well). The project's foundation is solid, though, it's based on this Udemy course I just completed.
